code/script.js

```diff
@@ -2,18 +2,110 @@
 
 // Step 1 - Welcome and introduction
 // Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+
+alert("Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.")
+let customerName = prompt("Please enter your name", "Your name")
+alert("Hi " + customerName + "! What are you craving today? \nPlease press \"ok\” to continue.")
+const errMsg = "Incorrect number, please enter 1, 2 or 3."
 
 // Step 2 - Food choice
 // Your code goes here
 
+let foodChoice = ""
+while (foodChoice == "") {
+  let choice = prompt("Please enter a number: \n 1 - Pizza \n 2 - Pasta \n 3 - Salad", "Please type you number of choice here")
+  if (choice == 1) {
+    foodChoice = "Pizza"
+  } else if (choice == 2) {
+    foodChoice = "Pasta"
+  } else if (choice == 3) {
+    foodChoice = "Salad"
+  } else {
+    alert(errMsg)
+  }
+}
+
+alert("You have selected: " + foodChoice + ". Yummy!")  
+
 // Step 3 - Subtype choice
 // Your code goes here
 
+let foodType = ""
+while (foodType == "") {
+  if (foodChoice == "Pizza") {
+    choice = prompt("Please enter a number: \n 1 - Napolitana \n 2 - Margherita \n 3 - Sicilian", "Please type you number of choice here") 
```

Comment:
It would be cool if you also included something in the message like, **"What type of pizza would you like?"** Please enter a number.....🤩